deposit/bytecode-verification/lemmas.k

```diff
@@ -182,21 +176,21 @@ module LEMMAS
     rule N &Int MASK => N  requires MASK ==Int (2 ^Int (log2Int(MASK) +Int 1)) -Int 1 // MASK = 0xffff...f
                             andBool 0 <=Int N andBool N <=Int MASK
 
-    // 0xff..ff00..00 (16 1's followed by 240 0's)
-    rule 115790322390251417039241401711187164934754157181743688420499462401711837020160 &Int #asWord(WS1 ++ WS2)
+    // 0xff00..00 (16 1's followed by 240 0's)
+    rule (pow256 -Int (2 ^Int 240)) &Int #asWord(WS1 ++ WS2)
```

Comment:
This does not work as expected. Function symbols are not evaluated in the LHS of function rules, only do macros. The fact that proofs pass, this lemma is not used at all.